src/demo-app/a11y/list/list-a11y.ts

```diff
@@ -0,0 +1,55 @@
+import {Component} from '@angular/core';
+
+@Component({
+  moduleId: module.id,
+  selector: 'list-a11y',
+  templateUrl: 'list-a11y.html',
+  styleUrls: ['list-a11y.css']
+})
+export class ListAccessibilityDemo {
+  items: string[] = [
+    'Pepper',
+    'Salt',
+    'Paprika'
+  ];
+
+  messages: any[] = [
+    {
+      from: 'Nancy',
+      subject: 'Brunch?',
+      message: 'Did you want to go on Sunday? I was thinking that might work.',
+      image: 'https://angular.io/generated/images/bios/julie-ralph.jpg'
+    },
+    {
+      from: 'Mary',
+      subject: 'Summer BBQ',
+      message: 'Wish I could come, but I have some prior obligations.',
+      image: 'https://angular.io/generated/images/bios/juleskremer.jpg'
+    },
+    {
+      from: 'Bobby',
+      subject: 'Oui oui',
+      message: 'Do you have Paris reservations for the 15th? I just booked!',
+      image: 'https://angular.io/generated/images/bios/jelbourn.jpg'
+    }
+  ];
+
+  links: any[] = [
```

Comment:
If you remove the `any[]` from these arrays, TS should be able to figure out the proper type.